go/src/koding/kites/kloud/machine/client.go

```diff
@@ -0,0 +1,62 @@
+package machine
+
+import "time"
+
+// Machine represents a single machine.
+type Machine struct {
+	Team      string    `json:"team"`
+	Stack     string    `json:"stack"`
+	Provider  string    `json:"provider"`
+	Label     string    `json:"label"`
+	IP        string    `json:"ip"`
+	CreatedAt time.Time `json:"createdAt"`
+	Status    Status    `json:"status"`
+	Users     []User    `json:"users"`
+}
+
+// Status represents the current status of machine.
+type Status struct {
```

Comment:
same for MachineStatus https://github.com/koding/koding/blob/master/go/src/koding/db/models/machine.go#L14